tests/test.ts

```diff
@@ -22,27 +22,27 @@ ensureCleanGeneratedFolder();
 // Generate the new baselines
 for (const fileName of fs.readdirSync('cases')) {
     describe("Generating baseline for " + fileName, () => {
-        const text = fs.readFileSync(path.join('./cases', fileName), 'utf8');
-        const parsedFileName = path.parse(fileName);
-        const { markerScopes, wholeBaseline } = build.generateScopes(text, parsedFileName);
-        if (markerScopes) {
-            addTestCase(parsedFileName.name + '.txt', markerScopes);
-        }
-        addTestCase(parsedFileName.name + '.baseline.txt', wholeBaseline);
+        it('Comparing generated', () => {
+            const text = fs.readFileSync(path.join('./cases', fileName), 'utf8');
+            const parsedFileName = path.parse(fileName);
+            const { markerScopes, wholeBaseline } = build.generateScopes(text, parsedFileName);
+            if (markerScopes) {
+                assertBaselinesMatch(parsedFileName.name + '.txt', markerScopes);
+            }
```

Comment:
Problem with this is that it wont generate both baselines if the first one doesn't match..

Reply:
Ok, I've move this back to two `it`s